v_3_3_4/types.go

```diff
@@ -15,6 +15,10 @@ type Params struct {
 	// DisableEncryptionAtREST flag when set removes all manifests from the cloud
 	// config related to Kubernetes encryption at REST.
 	DisableEncryptionAtREST bool
+	// DisableIngressController flag when set removes all manifests from the
```

Comment:
Nitpick: `DisableIngressController flag when set` doesn't read very nicely for me. But maybe thats just me. Or is there a comma missing? `when set, removes...`?

Reply:
Yes, I see what you mean. I added a full stop after flag so the new sentence describes what it does.